### ISSUE TITLE
Improving Frontend

### ISSUE THREAD

**Issue by ritwik12** [up-for-grabs/up-for-grabs.net#930]
At present, up-for-grabs.net is great content wise and is also good in front-end, But I can see a lot of potential for improvement as there are so many great web designers who can make up-for-grabs.net a very beautiful website. I would recommend [awwwards](https://www.awwwards.com/) for getting ideas.

Few points to work on as per @camilosampedro suggestions in the comments:

- [ ] Add task filter by last update (First of all active).
- [x] Сhange layout of the page, in particular, move filters to the side menu. Thereby solving the problem of unused space.
- [x] Add dark (night) mode.
- [ ] Fix dark mode contrast.
- [ ] Set nicer colors and fonts.

**Comment by NjalTheKnut** [comment]
I have an idea that might help tremendously.  I could attempt to implement a scrolling "back to top" button that would allow users who have scrolled way down the page to get back up to the search bar without needing to scroll all the way back manually.  Other suggestions may come to me as I use the site.

**Comment by ritwik12** [comment]
@NjalTheKnut That will be a good feature, go for it.

**Comment by sunilkumarvalmiki** [comment]
hi,can i undertake the work proposed by NjalTheKnut...

**Comment by NjalTheKnut** [comment]
I actually found out how to do it recently.  I am willing to do it and have a second person go over the code to ensure I am doing things in the best fashion possible.

Being somewhat inexperienced with remote team mechanics, I would appreciate a little guidance acquiring the code and making certain I don't foul up production branches in the VCS.

Also, am I allowed to introduce jQuery, if it is not already used in this project?

**Comment by sunilkumarvalmiki** [comment]
[up-for-grabs.net-gh-pages 2.zip](https://github.com/up-for-grabs/up-for-grabs.net/files/2246438/up-for-grabs.net-gh-pages.2.zip)
-------------------------------------------------------------

![img](https://user-images.githubusercontent.com/24809771/43476735-ed64f6a8-9516-11e8-8520-b0f60ba1e00b.png)
------------------
i have completed the task...

**Comment by ritwik12** [comment]
@sunilkumarvalmiki  Can you host this on your fork? it will be easy to review that way.

**Comment by sunilkumarvalmiki** [comment]
yes, i forked it to my profile.

**Comment by ritwik12** [comment]
@NjalTheKnut I meant if you can host this as github pages on your fork so that we can see it live.

**Comment by Supreme-Kai** [comment]
@ritwik12 Hey there! I actually wanted to make some changes to the front-end, but I noticed the _site folder holding the index.html and stylesheets is in git.ignore. Just wanted to know if I can "un-ignore" the file or if not, how I can proceed.

**Comment by shiftkey** [comment]
@Supreme-Kai 

> Hey there! I actually wanted to make some changes to the front-end, but I noticed the _site folder holding the index.html and stylesheets is in git.ignore. Just wanted to know if I can "un-ignore" the file or if not, how I can proceed.

Those files are not supposed to be committed, as they are regenerated by [Jekyll](https://jekyllrb.com/) - the engine underneath GitHub Pages.

If you want to test things out on your fork, [this guide](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/) should help ensure it is configured right. Alternatively, you can try and get Jekyll [running locally](https://jekyllrb.com/docs/quickstart/) or use the included `Dockerfile` in this repository.

**Comment by Supreme-Kai** [comment]
@shiftkey  Hey thanks for responding. I had the environment running locally already, I was just testing the index and stylesheet from the _site folder and so naturally my changes weren't showing when I ran `git status`. My fault. That said can I be assigned to this? I have some ideas on how to tweak this style wise.

**Comment by shiftkey** [comment]
> That said can I be assigned to this? I have some ideas on how to tweak this style wise.

@Supreme-Kai the assignee field is based on organization membership, but feel free to work on this and propose changes in a PR

**Comment by Supreme-Kai** [comment]
Oh right, got it. Awesome, I'll do that. Thanks.

**Comment by ritwik12** [comment]
<img width="1630" alt="Screenshot 2020-04-05 at 1 43 18 PM" src="https://user-images.githubusercontent.com/20038775/78509543-75bc2480-7743-11ea-843f-e8643bf220f6.png">

These two white spaces at the sides always trouble me :P 
Let's use that empty space a lit bit to improve readability.

How about if we just increase the Centre block on both sides so that The projects with Long description will at least get in few lines than so many lines as shown in this image.

Also, we can improve the button on the bottom right corner to a good design. For now, it is simply a bold look.

**Comment by rustmn** [comment]
> These two white spaces at the sides always trouble me :P
> Let's use that empty space a lit bit to improve readability.
> 
> How about if we just increase the Centre block on both sides so that The projects with Long description will at least get in few lines than so many lines as shown in this image.
> 
> Also, we can improve the button on the bottom right corner to a good design. For now, it is simply a bold look.

@ritwik12 I have some ideas for improving FE of the project a and enhance user-experience.

**Comment by xenoworkforce** [comment]
@ritwik12 I have some ideas for improving FE of the project and enhance user-experience:

1. Add task filter by last update (First of all active).
2. Сhange layout of the page, in particular, move filters to the side menu. Thereby solve the problem of unused space.
3. Add night-mode (many people do not like to look at the white screen).
4. Set nicer colors and fonts.

I can write this in TypeScript using react / next.js, if you will allow. (I know that using a framework for such a simple project can be overkill but maybe he will grow into something more?)

**Comment by ritwik12** [comment]
@rustmn I think it will be good. Can you please share a design or something to see how that would look like before actually making it.
@shiftkey can you suggest the improvements for the front end that we would want to see on the site.

**Comment by xenoworkforce** [comment]
@ritwik12 ok, I will make PSD layout.

**Comment by xenoworkforce** [comment]
@ritwik12 okay... something like this. Of course, design is not final and i'm open for your suggestions. Also, current logo is not suitable for the dark theme, if you have .eps / .ai or .svg of it, i can fix this.
![MacBook](https://user-images.githubusercontent.com/28675457/79380496-35da1600-7f69-11ea-97be-0cb837718e91.jpg)

**Comment by ritwik12** [comment]
@rustmn That's good. From my point of view I would love to have the old look and improve that. The current look is awesome and lets improve that. For Dark mode. It will be good if we have a toggle or button to switch between Dark and Light modes of the site. Will prefer the Light mode for default though.

I really like the left part, Your views @shiftkey for left side area.

**Comment by ritwik12** [comment]
@rustmn If it's not a long work, can you try and add the left part to the present website and share how it looks. Check the tone of color for whichever looks better there. I guess black would suite too

**Comment by xenoworkforce** [comment]
@ritwik12 ok, done. It might look something like this:
view - 31.131.27.205:4000
changes - https://github.com/rustmn/up-for-grabs.net/commit/7697b7397b1661dd4b9a087fc2646f33d9a27ed9 

The search does not work correctly yet, because I have not figured out how it works.

Also, when I open your site, my screen locks for 2 seconds, due to the fact that all projects are rendered at the same time. I can fix this. If this is done for web scrapers, I can make api for this.

**Comment by ritwik12** [comment]
@rustmn Not able to see the view :(

**Comment by xenoworkforce** [comment]
@ritwik12 hm... strange.

Then so: https://rustmn.github.io

**Comment by ritwik12** [comment]
@shiftkey This is a raw structure, can you suggest what to use and improve in this  https://rustmn.github.io 

Also, if you have some design to improve the Front end. Please share.

**Comment by ritwik12** [comment]
@rustmn Thanks a lot. For now left side looks pretty good but isn't matching with the current design :( 

Also, the right side is stretched a lot. better keep the original only.

**Comment by ritwik12** [comment]
@Jai2305 Thanks for the contributions. I wanted to change the current Look of the site which is mostly focused at the center. If you are interested. Please have a look at that.

**Comment by Jai2305** [comment]
Hi, @ritwik12  first of sorry for late reply, firstly  if you have scrolled down to say 50 projects , u have to come all the way up again to use different filters (I know there is a button  Scroll to Top) but still the website should have a fixed navigation bar or menu which stays on top ( let me know your opinion ) , secondly as you said currently now the website is just displaying content in the center leaving too much space on the left and right , which is not my personal preference as in web design , but as we know website is made using <b> jekyll </b> and is using a template for rendering pages, now in my opinion what we can do is  
* use another template  <b>or</b>
* make changes in / customize  the current template ( which seems to be much more safe than the previous one )
let me know your thoughts

**Comment by ritwik12** [comment]
@Jai2305 Makes sense for having to scroll up all the time. it is always better to have Menu bar or pagination at least. 
we can use another template also and customize the current one too. Though a lot of automated stuff is currently going on with the current template so changing it would mean a lot of work. It is better if we just change the styling and all to get it done with ease.

**Comment by Jai2305** [comment]
Then we should customize the current template , I will try to come up with a design , it may take a while though, so I guess should make this open for everyone interested , if anyone comes up with the a front end design then they can post a message here or open a pull request and tag this issue within, so that people who are interested can work together

**Comment by ritwik12** [comment]
@Jai2305 Sure, please take your time. yes this is open to everyone. Any can contribute and work together.

**Comment by elvinuchendu** [comment]
Hi, is this issue still ongoing? I would like to take part.

**Comment by ritwik12** [comment]
@elvinuchendu Yes, for now @Jai2305 showed interest and if you have any ideas. We would be happy if you can contribute.

**Comment by elvinuchendu** [comment]
> @elvinuchendu Yes, for now @Jai2305 showed interest and if you have any ideas. We would be happy if you can contribute.

Awesome! Are you able to assign this to me?

**Comment by ritwik12** [comment]
@elvinuchendu This Issue consists of multiple parts so won't be able to assign as others might be working too. but since you called out here. Consider it assigned. Just call out what you are working on so that someone else will not be working on the same at this time.

**Comment by ghost** [comment]
> @ritwik12 I have some ideas for improving FE of the project and enhance user-experience:
> 
> 1. Add task filter by last update (First of all active).
> 2. Сhange layout of the page, in particular, move filters to the side menu. Thereby solve the problem of unused space.
> 3. Add night-mode (many people do not like to look at the white screen).
> 4. Set nicer colors and fonts.
> 
> I can write this in TypeScript using react / next.js, if you will allow. (I know that using a framework for such a simple project can be overkill but maybe he will grow into something more?)

You are definitely right about the dark mode option. IMO, I think a dark mode option should be the default one.

**Comment by ghost** [comment]
By the way, on the dark mode, the contrast really isn't that great (especially for the logo).
![image](https://user-images.githubusercontent.com/61570792/146692812-f500a292-7e7d-49f2-ba0d-cdb082a4030b.png)

**Comment by ritwik12** [comment]
@ConnorAhern Those points are good to have, I have added those in the first comment of this issue as action items.

**Comment by ritwik12** [comment]
@ConnorAhern If you are up for working on this, please feel free to drop samples of the work or PR.

**Comment by ritwik12** [comment]
> By the way, on the dark mode, the contrast really isn't that great (especially for the logo). ![image](https://user-images.githubusercontent.com/61570792/146692812-f500a292-7e7d-49f2-ba0d-cdb082a4030b.png)

@ConnorAhern  Yeah, it needs to be worked on. Thanks for the nudge.

**Comment by ghost** [comment]
> @ConnorAhern If you are up for working on this, please feel free to drop samples of the work or PR.

Alright!

**Comment by ghost** [comment]
Do we want the layout to look like this as @rustmn mentioned? I could start on that, but we'd need to get other PRs merged first so that there aren't merge conflicts.

![image](https://user-images.githubusercontent.com/61570792/146810927-ebb2a25c-4dfb-487a-9757-e0c08b277c7e.png)

**Comment by ritwik12** [comment]
@ConnorAhern This is one of the good ideas, if you have other ideas it would be good to go with them. if you don't want to start from scratch and this looks good to you, it should be fine too.

**Comment by ghost** [comment]
> @ConnorAhern This is one of the good ideas, if you have other ideas it would be good to go with them. if you don't want to start from scratch and this looks good to you, it should be fine too.

I'd be happy working on implementing that design. It looks great to me.

**Comment by ritwik12** [comment]
Go with it please @ConnorAhern

**Comment by ghost** [comment]
Alright. I can start with it today.

**Comment by ghost** [comment]
> Do we want the layout to look like this as @rustmn mentioned? I could start on that, but we'd need to get other PRs merged first so that there aren't merge conflicts.
> 
> ![image](https://user-images.githubusercontent.com/61570792/146810927-ebb2a25c-4dfb-487a-9757-e0c08b277c7e.png)

Does anyone know what font is used in this?

**Comment by ritwik12** [comment]
Current up-for-grabs.net font is good, lets keep it that way for now.

**Comment by ghost** [comment]
> Current up-for-grabs.net font is good, lets keep it that way for now.

Alright

**Comment by ghost** [comment]
Can we do some kind of survey of the users of the site and ask if they prefer light mode or dark mode as a default?

**Comment by ritwik12** [comment]
Yeah, we can check that, Do we have a way to check how many users click on the dark mode? it toggles a theme that is static I guess, I am not sure if we get the reply from there or it is only on the user side. If it is dynamic then we can surely record that and get an estimate about how many users are using dark mode or light mode.

**Comment by ghost** [comment]
> Yeah, we can check that, Do we have a way to check how many users click on the dark mode? it toggles a theme that is static I guess, I am not sure if we get the reply from there or it is only on the user side. If it is dynamic then we can surely record that and get an estimate about how many users are using dark mode or light mode.

We could maybe set up something with Google Analytics? I think it's possible to send custom data to Google Analytics.

**Comment by ghost** [comment]
I have made some progress. See #3023.

**Comment by ghost** [comment]
> I have an idea that might help tremendously. I could attempt to implement a scrolling "back to top" button that would allow users who have scrolled way down the page to get back up to the search bar without needing to scroll all the way back manually. Other suggestions may come to me as I use the site.

By the way, I accidentally removed this in my changes. I will add it back in a bit.

**Comment by ghost** [comment]
> > I have an idea that might help tremendously. I could attempt to implement a scrolling "back to top" button that would allow users who have scrolled way down the page to get back up to the search bar without needing to scroll all the way back manually. Other suggestions may come to me as I use the site.
> 
> By the way, I accidentally removed this in my changes. I will add it back in a bit.

It is back.

**Comment by Jai2305** [comment]
Hi , been a long time, was really busy with work...  
so I had a simple to execute yet pretty design in mind 
![image](https://user-images.githubusercontent.com/47395196/159920655-6deee23b-410c-4e4a-928e-8a73e1239274.png)
So I was thinking maybe we can use the space here in a more efficient manner by displaying projects in form of tabs/cards , the design would be responsive and they will be of the same dimension , the height here can be managed by displaying a read more option, if you like the idea then I can try to work further on it and make this design better and prettier

**Comment by Jai2305** [comment]
we can play with the colors and see what seem fit in dark mode too , also for menu , how about a toggle button on the side which will reveal a banner for filters , it's just an idea , I have not worked on it yet .

**Comment by ritwik12** [comment]
@Jai2305 Thanks for the ideas. Tabs looks good to me. The SS is not having tabs of same size but I am assuming you meant all will be of similar size and this is just a example without same size.

We can also have the toggle button for filters but if there is space available then the current one seems fine too.

**Comment by Jai2305** [comment]
yeah in the SS they are not same size because I have not worked on it yet ; to be honest , I am not sure if read more would be a good choice that's why I was thinking about css grid auto-placement , after that it won't waste any space tabs will cover up the available space , if I make any progress , I will make  sure to share here first before working further .

**Comment by Jai2305** [comment]
Hi, I made a pull request #3146 , but the checks are failing on that , can someone suggest me reasons due to which they might be failing as well as the changes I can make in the code , also please comment on the look in the pull request so that I know it is worth working on it or it should be changed , any ideas will be appreciated

**Comment by ritwik12** [comment]
@Jai2305 Those are due to merge conflicts. If you see stuff like <<<<< and ======= in your code when you commit something and create a pull request. that is merge conflict. Reason for that is due to files having a conflict between two changes.

Suppose you made a change in some file in your local and did not yet pushed it by creating a PR. Someone else also made some changes in their local and pushed it by creating a PR. So, now when you create a PR for those same files, there will be changes which might be conflicting as the file in the upstream (i.e main repo) is already changed.
 
```
<<<<<<< HEAD
this is some content to mess with
content to append
=======
totally different content to merge later
>>>>>>> new_branch_to_merge_later
```

**Comment by ritwik12** [comment]
Try to read https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts. For simple steps, github opens a merge conflict window when there are merge conflicts. just go there and remove those lines with <<<<< and ===== which are not related to your code.

**Comment by Jai2305** [comment]
Yeah, actually I anticipated that checks are failing due to merge conflicts but they can only be resolved if I get yours or any authorities to take a look at it and let me know that changes I have made are acceptable or not , because for eg , I changed the table tags to div tags for better front end design because there is not so much you can do while using table tags , the changes are very small in terms of number of lines changed , I think changing the **table** tag into the **div** is the only major change which requires you/authorities to take a look otherwise there are some minor changes like some lines in CSS file and so on . 

In all let me know what you think , and what do you think of the new design , I was thinking that after it gets merged , we can do some more changes after that , but majorly the design will look something like this( depicted in screenshots ) 
Thanks for replying @ritwik12

**Comment by Bencourt** [comment]
Hi there, I posted a comment on a different issue thread about updating the logo, but it seems like this thread has more attention so I thought I might post again. I have come up with a couple alternate logo ideas since the current logo has issues with readability in dark mode. I was hoping to get some thoughts and propose changing the current logo to one of my designs. Please let me know what you all think!
![up_for_grabs_logo_white](https://user-images.githubusercontent.com/9601471/166167854-e997e26c-7d03-440e-9ccf-c4d50d421cd1.svg)
![up_for_grabs_logo](https://user-images.githubusercontent.com/9601471/166167855-bda359af-8d47-404f-9351-78a5e0e79569.png)

**Comment by Bencourt** [comment]
Hi once again, I noticed some issues with the dark and light mode styling such as the header backgrounds having a white background in the dark mode, the return to top button being light in dark mode, the blue font color being somewhat difficult to read in dark mode, and the box drop shadow being somewhat visually confusing, so I made some updates to the styling and they currently live on my fork of the repo. I feel like these are good changes and they pass CI, but I haven't made a pull request because I am using the logo I redesigned so let me know if they are changes that should be pulled and I'll replace the logo and make a pull request.
![contribution2-end](https://user-images.githubusercontent.com/9601471/166396102-b54c98c5-b9ed-4aa6-b3c9-2f115278af52.PNG)
![contribution2-end-dark](https://user-images.githubusercontent.com/9601471/166396111-38f8233a-c22d-4328-bfbe-0f376c339b0b.PNG)

**Comment by ritwik12** [comment]
@Bencourt Thanks a lot for creating PR and contributing. I can't comment on the logo @shiftkey might be able to tell better.
I agree with you for Dark mode issues, can you please share Screenshot of the design after your changes?

**Comment by AllanOcelot** [comment]
Coming back after a long hiatus here, but I had a spare hour to crack this up whilst waiting for an email reply! 

Happy to share the Adobe XD file with anyone who wants it - just shoot me a mail. 
Sadly I lack the free time to code any of this currently but happy to discuss points with anyone interested. 
![Web 1920 – 1](https://user-images.githubusercontent.com/11334905/167298939-60999d68-22ba-438d-b014-f9feee449a10.png)

**Comment by Jai2305** [comment]
Hi, that's a really good design , i think it is better than the current, also I guess we can make the side menu bar collapse into a little burger option on top which will be set to sticky  so that user can use filters whenever they want after scrolling , all good but I guess we gotta keep the logo out of menu bar , if it collapses with the rest of the menu into a button it won't be visible at the website anymore or is it fine ?

**Comment by ritwik12** [comment]
That's a good design @AllanOcelot. Thanks for the ideas. @Jai2305  You are right, it would be great to have it as a collapsible side bar. For the logo I would say let's try to keep it on top left for this where after collapsible it'll stay there in form of a semi circle or circle.

Thoughts?

**Comment by Jai2305** [comment]
Yeah, I guess we can do that , I am just trying to figure out the mobile version of this in which rather then left logo stays on top.

**Comment by AllanOcelot** [comment]
Glad you like it - happy to tweak things, if there's interest in getting this going forward I can certainly Mock up a mobile version with a collapsible sidebar & smaller logo/icons etc. 

Realistically it would be Tuesday/Wednesday night due to work commitments - but I'm following the thread and will work other suggestions etc in

**Comment by ritwik12** [comment]
Yeah @AllanOcelot, that would be awesome. If you are busy and @Jai2305 is ready to work on it. It would be great to have the mobile design too.

**Comment by p4nu** [comment]
I'm also interested in implementing mobile layout when the design is ready 👍

**Comment by jalaniz1** [comment]
Another source of ideas for improvement could potentially come from looking at the Lighthouse report for up for grabs.net. I suppose these might be considered to be optimizations rather than improvements but thought of sharing anyway.

Here's a sample report run from my browser with mobile as the targeted platform:
[up_for_grabs_net.pdf](https://github.com/up-for-grabs/up-for-grabs.net/files/9033402/up_for_grabs_net.pdf)
![image](https://user-images.githubusercontent.com/1664815/177011651-9a6fb005-7e5b-4e6f-b7af-5d109298465a.png)

**Comment by ritwik12** [comment]
@jalaniz1 Thanks for sharing the report. I think we can work on this and open up few issues to take the concerns.

**Comment by JeetPatel1016** [comment]
I've been going through this thread and while I do not wish to greatly alter the layout of the site (it seems alternate layouts have been proposed by others), there are few changes that I made for the color theme and I'd like to suggest them.

Light Mode:
Some slight modifications have been made to the light theme and that includes:
- Contrast between page background and the Main container
  - Did only slight contrast as more contrast would only mess with the logo
- Typographical Hiearchy
  - Labels in the filter section were hard to read due to their comparatively small size
- Alignment of Project Cards with Filter Section
- Button Style so that the button elements sit well with the overall page

Preview:
![upforgrablight](https://user-images.githubusercontent.com/60153965/191255473-30014781-43c7-47df-8fd1-1b8d43d9c603.png)

Dark Mode:

- So rather than sticking to neutral shades of black, I used shades of blue in the palette to better couple with the light theme.
- Included [this](https://github.com/up-for-grabs/up-for-grabs.net/issues/930#issuecomment-1115553487) suggestion by @Bencourt on Heading banners
- Fixed Contrast Issues in Filter labels and Project Titles
- Added background color to the project cards to better differentiate from the container background

![upforgrabdark](https://user-images.githubusercontent.com/60153965/191261633-5176c012-3310-4a4d-a498-5436848ab706.png)

Please let me know about these changes and I'd appreciate further suggestions from your end.

**Comment by ritwik12** [comment]
@JeetPatel1016 Thanks for your suggestions. All your suggestions look good to me, especially the Dark Mode contrast. Please go ahead with your changes except for the logo change. The current logo in bright mode looks better than your suggestion here.
Please raise a PR.

**Comment by JeetPatel1016** [comment]
@ritwik12 I've raised a PR and dropped the logo change as you mentioned.

**Comment by tagener-noisu** [comment]
Not sure if I shoud create a separate issue for this. The first time I opened the [https://up-for-grabs.net/](https://up-for-grabs.net/) I couldn't understand where should I click to start. It's not intuitive enough that the "I want to get envolved" group is clickable.

**Comment by UsmanAhmadSaeed** [comment]
I'd love to take on the dark mode feature. Where can I find the project structure to know where to make changes?

**Comment by ffshreyansh** [comment]
@ritwik12  Hey I'd love to contribute in frontend mostly in Design and some features as I guess there is lots of room for changes, some design I thought along with the color combi. , let me know if this would work for the issue or not .
I just created this idea with some color and fonts layout changes.

![UPG](https://user-images.githubusercontent.com/51352120/205128007-1793434a-3a8a-4247-83f8-700cc18a81cf.png)

**Comment by jitendraKhadoliya** [comment]
BEFORE ==>

mistakes that I found =>

- responsive issue
- Icons are very big for the mobile screen
- icon pattern was not matching the scroll-up button was square and the theme button was in the circle 
- The main issue with the icons was the overlaying  on the text field which makes the user experience bad

![127 0 0 1_4000_(iPhone SE) (1)](https://user-images.githubusercontent.com/113577106/226391142-879cb462-a8a0-416b-97e9-4918066f6d23.png)

AFTER ==>

- added media query
- now the web app looks more UX friendly
- the same size of both buttons
- same height from the top and bottom 
- buttons are on the same line which looks more impressive
- margin right is also the same
- solved the problem of the overlay on the text field
![127 0 0 1_4000_(iPhone SE)](https://user-images.githubusercontent.com/113577106/226391250-5d68f069-d8c3-4371-8b51-2eea0e84b00d.png)

**Comment by Manan50** [comment]
@ritwik12 are there any more issues left for me to undertake as I am interested in improving the FE of site and contributing

**Comment by Murdock9803** [comment]
Hello @ritwik12, since its been a long time this issue has opened from, is there any help I can provide on this at present ? I would love to contribute :raised_hands:

**Comment by Mohd-Hamza-Khan** [comment]
Hello @ritwik12, I see the UI, since its been a long time this issue has opened

I believe up-for-grabs.net could benefit from a few UI improvements to enhance its look and usability.
Here are some ideas:

1) Cursor Animation, Adding a smooth cursor animation can give the site a modern touch.
2) Color Scheme, A well-chosen color palette will make the design more cohesive and appealing.
3) Updated Menu Bar, A stylish and responsive menu bar can improve navigation.
4) Choosing fonts that work well together will create a polished, professional feel.

I would love to contribute, here is my suggestion to add this type of animation
![1](https://github.com/user-attachments/assets/31e05f7c-6016-43b5-84f8-905061d45522)
![2](https://github.com/user-attachments/assets/fadaaf86-8d68-49be-80ec-3b82e5f8289e)
![3](https://github.com/user-attachments/assets/4e7fc006-c094-48a1-a41f-a1a3a9e5ae0e)

These changes would make the site both visually pleasing and more user-friendly

**Comment by Mohd-Hamza-Khan** [comment]
I want to add these type of changes, please assign to me!

**Comment by Noodlebox1** [comment]
The dark mode as it is currently on the site looks great, but I noticed that:
1. even if your system preferences are set to dark, the site will still load as light
2. if you reload the page or open it in a new tab, it doesn't remember your theme preference
I know the site was primarily designed with light mode in mind, but it'd be nice to fix these issues.
